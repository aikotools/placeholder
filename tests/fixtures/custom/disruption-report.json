{
  "reportId": "{{gen:uuid:disruption-2025-042}}",
  "title": "Verspätungsmeldung - {{train:icenumber:517}}",
  "timestamp": "{{time:calc:0:seconds}}",
  "trains": [
    {
      "trainNumber": "{{train:icenumber:517}}",
      "type": "{{train:type:ICE}}",
      "currentStation": "{{train:station:Frankfurt Hbf}}",
      "delay": "{{train:delay:25}}",
      "platform": "{{train:platform:12}}",
      "status": "delayed",
      "passengers": "{{gen:number:342}}"
    },
    {
      "trainNumber": "{{train:icenumber:891}}",
      "type": "{{train:type:IC}}",
      "currentStation": "{{train:station:Köln Hbf}}",
      "delay": "{{train:delay:12}}",
      "platform": "{{train:platform:8}}",
      "status": "delayed",
      "passengers": "{{gen:number:198}}"
    },
    {
      "trainNumber": "{{train:icenumber:234}}",
      "type": "{{train:type:ICE}}",
      "currentStation": "{{train:station:Hamburg Hbf}}",
      "delay": "{{train:delay:0}}",
      "platform": "{{train:platform:5}}",
      "status": "on-time",
      "passengers": "{{gen:number:412}}"
    }
  ],
  "summary": {
    "totalTrains": "{{gen:number:3}}",
    "delayedTrains": "{{gen:number:2}}",
    "onTimeTrains": "{{gen:number:1}}",
    "averageDelay": "{{gen:number:12|toNumber}}",
    "reportGenerated": "{{time:calc:0:yyyy-MM-dd HH\\:mm\\:ss}}"
  }
}
