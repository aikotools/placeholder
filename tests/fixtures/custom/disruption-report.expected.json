{
  "reportId": "disruption-2025-042",
  "title": "Verspätungsmeldung - ICE 517",
  "timestamp": 1742040000,
  "trains": [
    {
      "trainNumber": "ICE 517",
      "type": "ICE",
      "currentStation": "Frankfurt Hbf",
      "delay": 25,
      "platform": 12,
      "status": "delayed",
      "passengers": 342
    },
    {
      "trainNumber": "ICE 891",
      "type": "IC",
      "currentStation": "Köln Hbf",
      "delay": 12,
      "platform": 8,
      "status": "delayed",
      "passengers": 198
    },
    {
      "trainNumber": "ICE 234",
      "type": "ICE",
      "currentStation": "Hamburg Hbf",
      "delay": 0,
      "platform": 5,
      "status": "on-time",
      "passengers": 412
    }
  ],
  "summary": {
    "totalTrains": 3,
    "delayedTrains": 2,
    "onTimeTrains": 1,
    "averageDelay": 12,
    "reportGenerated": "2025-03-15 12:00:00"
  }
}
